electrum/tests/test_mpp_split.py

```diff
@@ -0,0 +1,75 @@
+import random
+
+import electrum.mpp_split as mpp_split  # side effect for PART_PENALTY
+from electrum.lnutil import NoPathFound
+
+from . import ElectrumTestCase
+
+PART_PENALTY = mpp_split.PART_PENALTY
+
+
+class TestMppSplit(ElectrumTestCase):
+    def setUp(self):
+        super().setUp()
+        random.seed(0)  # split should only weakly depend on the seed
+        # test is dependent on the python version used, here 3.8
+        # undo side effect
+        mpp_split.PART_PENALTY = PART_PENALTY
```

Comment:
Would it not make more sense to do this in `tearDown`, after (instead of before) each test?
It probably does not matter right now but note that other tests (in other files) might now see a non-default `mpp_split.PART_PENALTY`; which would not be the case if this was moved to `tearDown`.